kubernetes/check.py

```diff
@@ -496,10 +496,11 @@ def _update_kube_events(self, instance, pods_list, event_items):
             tags = self.kubeutil.extract_event_tags(event)
             tags.extend(instance.get('tags', []))
 
-            title = '{} {} on {}'.format(involved_obj.get('name'), event.get('reason'), node_name)
+            title = '{} {}'.format(involved_obj.get('name'), event.get('reason'))
             message = event.get('message')
             source = event.get('source')
             if source:
+                title += ' on {}'.format(source.get('host', ''))
```

Comment:
That's great, thanks for fixing it.

Could you also use it for the `host` field here please? https://github.com/wpalmeri/integrations-core/blob/290fb1827f30ed4d2c0478e0b6ca077b4e29f5e4/kubernetes/check.py#L508

You could just replace L501 with `source = event.get('source', {})`
and then do
```python
'host': source.get('host', '')
```

The aggregator will override the hostname if it's blank anyway.